.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: python
+
+python:
+  - '2.7'
+
+sudo: required
+
+services:
+  - docker
+
+install:
+  - docker pull f5devcentral/containthedocs
+
+script:
+  - ./scripts/docker-docs.sh ./scripts/test-docs.sh
+
+deploy:
+  - provider: script
+    skip_cleanup: true
+    on:
+      branch: master
+      repo: F5Networks/f5-neutron-lbaas-dashboard-docs
+    script:
+      - ./scripts/deploy-docs.sh publish-cloud-docs-to-prod openstack/dashboard v9
```

Comment:
In this script, "openstack/dashboard" defines the path in clouddocs.f5.com website where your documentation will publish. Are you sure you want that to be `clouddocs.f5.com/cloud/openstack/dashboard/v9`?

I recommend following the same format as the lbaasv2-driver, which would be:
`      - ./scripts/deploy-docs.sh publish-cloud-docs-to-prod openstack/lbaasv2-dashboard v9`

Reply:
agree